style.css

```diff
@@ -621,3 +621,334 @@ hr {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-evenly;
+    align-items: center;
+    gap: 15px;
+  }
+
+  .card-container:nth-child(even) {
+    align-items: center;
+  }
+
+  .socials li img {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    margin-top: -2rem;
+    color: #505f79;
+  }
+
+  .techs ul li img {
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    text-align: center;
+    padding: 0.2rem;
+  }
+
+  .social li a img {
+    margin-right: -1rem;
+    color: #505f79;
+  }
+
+  .gb-img img:nth-child(even) {
+    width: 100%;
+  }
+
+  .card-containers .gb-img img {
+    width: 100%;
+  }
+
+  .about {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 5rem;
+    justify-items: center;
+    align-items: baseline;
+    padding-top: 8rem;
+    padding-left: 6%;
+    padding-right: 6%;
+    padding-bottom: 5rem;
+  }
+
+  .about-h1-b {
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+    line-height: 52px;
+  }
+
+  .techs {
+    width: 100%;
+  }
+
+  .about-p-b {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .under-lines {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #344563;
+    padding-bottom: 0.9rem;
+    border-bottom: 2px solid #dfe1e6;
+    padding-top: -2rem;
+  }
+
+  .heading h3 {
+    font-size: 35px;
+  }
+
+  .techs ul {
+    display: flex;
+    width: 30%;
+    padding: 0.5rem;
+  }
+
+  .techs ul li {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    padding: 1.5rem;
+    font-weight: 500;
+    font-size: 12px;
+  }
+
+  .socials li a {
+    margin-right: -1.5rem;
+    width: 20px;
+  }
+
+  .solid {
+    transform: translateY(-150%);
+    display: flex;
+    justify-content: space-between;
+    cursor: pointer;
+    margin-bottom: 4rem;
+  }
+
+  .form_container {
+    overflow: hidden;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+  }
+
+  .form {
+    padding: 5%;
+    color: #fff;
+    background-image: url("img/contact\ form\ background\ shapes.svg");
+    background-color: #6070ff;
+    background-repeat: no-repeat;
+    border-radius: 5rem 0 0 0;
+    background-position: -5px 110px;
+    background-size: contain;
+  }
+
+  .form_container .p {
+    width: 40%;
+    text-align: center;
+    transform: translateY(17%);
+  }
+
+  .form_heading {
+    padding-top: 5rem;
+    transform: translateY(20%);
+    font-weight: 700;
+    font-size: 40px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+  }
+
+  form input {
+    border-radius: 0;
+    padding: 0.8rem;
+    width: 150%;
+    color: #172b4d;
+    outline: none;
+    border: 0;
+    margin-bottom: 1rem;
+    font-weight: 400;
+    font-size: 18px;
+    align-items: center;
+    transform: translateX(-18%);
+  }
```

Comment:
- [ ] Good job with your contact section, could you please add a `border-radius` to the button below your form? This will help it match the [given template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122) perfectly. 🎯